resources/zib/zib-PharmaceuticalProduct.xml

```diff
@@ -0,0 +1,176 @@
+      <alias value="FarmaceutischProduct" />
+      <mapping>
+        <identity value="zib-pharmaceuticalproduct-v2.1.2-2020EN" />
+        <map value="NL-CM:9.7.19926" />
+        <comment value="PharmaceuticalProduct" />
+      </mapping>
+    </element>
+    <element id="Medication.extension">
+      <path value="Medication.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Medication.extension:description">
+      <path value="Medication.extension" />
+      <sliceName value="description" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PharmaceuticalProduct.Description" />
+      </type>
+    </element>
+    <element id="Medication.code">
+      <path value="Medication.code" />
+      <short value="MedicationCode" />
+      <definition value="Coding medication in the Netherlands is done on the basis of the G standard (issued by Z-index), which is filled under the direction of KNMP. &#xD;&#xA;&#xD;&#xA;The coded medication can be expressed as: &#xD;&#xA;* GTIN International Article Number&#xD;&#xA;* ZI-nummer (2.16.840.1.113883.2.4.4.8)&#xA;* Trade product code (HPK)&#xA;* Prescription code (PRK)&#xA;* Generic product code (GPK)&#xA;* Anatomic Therapeutic Classification code (ATC)&#xA;* Substance Name Code (SNK)&#xA;*  Substance Name Code, in combination with Route of Administration (SSK)&#xA;*  SNOMED CT code&#xD;&#xA;&#xD;&#xA;The GTIN enables identification of the product including its origin with a barcode. The ZI number represents the tradeproduct and the package size. The HPK is the code for the trade product (with the brand name) as used per dose/per time the medication is taken (1 pill, 1 puff, 1ml) The PRK contains als GPK information supplemented with information needed to prescribe the right product. this level is intended to facilitate generic prescription of drugs. The GPK defines the pharmaceutical characteristics of a product: ingredients, strengths, units, pharmaceutical form and route of administration. The Substance Name Code (SNK) is the (active) compound. Further information about the G-Standaard levels see https://www.z-index.nl So called 90.000.000 numbers are used in local IT systems. These numbers shall not be used for external communication. In the this is necessary only the description (not the code) may be included. This is in accordance with national agreements (Nictiz Standard)." />
+      <alias value="ProductCode" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.12--20200901000000" />
+      </binding>

```

Comment:
This valueSets has a reference to other valueSets and the most binding strength has a value "required". The **IngredientCodeGTINCodelist** has a binding strength of "Extensible". How do we modulate this?

Reply:
This will likely become required for all. Therefore I would say we move forward with the required binding.